changelog-versions/9.0.0.md

```diff
@@ -36,3 +36,14 @@
 - Lists [Hover and pressed states] [#1204](https://github.com/Telefonica/mistica-design/issues/1204)
 - Navigation bars logo interaction in web | Update specs [#1267](https://github.com/Telefonica/mistica-design/issues/1267)
 - → Text constant project | Design documentation [#824](https://github.com/Telefonica/mistica-design/issues/824)
+
+
+## Migration notes
+
+### Autolayout changes
+
+We've evolved the auto-layout of the components to have the strokes included, which means that the height and width of multiple components in the library will increase the border width x2. In those artboards where the design is not using auto-layout, the spacing between element will be affected (please review those cases
```

Comment:
this "x2" is scary 😄. Maybe is easier to say that all components with borders will be increased 2px in height?

Reply:
Agree :) Changed